plugins/woocommerce/src/StoreApi/Routes/V1/CheckoutOrder.php

```diff
@@ -177,6 +177,14 @@ protected function get_route_post_response( \WP_REST_Request $request ) {
 		);
 	}
 
+	/**
+	 * Since this endpoint only operates on existing orders, we don't need to do updates based on
+	 * the cart data.
+	 *
+	 * @param \WP_REST_Request $request Request object.
+	 */
+	protected function cart_updated( \WP_REST_Request $request ) {}
```

Comment:
Calling  the parent `cart_updated` method in theory shouldn't have any negative side effects since the method updates the cart using the data from the draft order, not the order currently being processed by Checkout Order, however, I think that adds unnecessary overhead, but please let me know what do you think.